src/lib/state.ts

```diff
@@ -241,19 +267,19 @@ export const dumpNode = (
     status: node.status,
     ready: node.ready,
     isListening: node.isListening,
-    isFrequentListening: node.isFrequentListening,
+    isFrequentListening: Boolean(node.isFrequentListening),
```

Comment:
Why wrap it in boolean, if it's undefined, we should put in `null` to indicate we don't know

Reply:
The type used to be a `boolean`, I think we always returned either `true` or `false`

Reply:
You were right. Updated